homeassistant/components/vallox/strings.json

```diff
@@ -8,10 +8,19 @@
         "data_description": {
           "host": "Hostname or IP address of your Vallox device."
         }
+      },
+      "reconfigure": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "Hostname or IP address of your Vallox device."
```

Comment:
Is it possible to reuse this string from the `user` step?

Reply:
Yes, use `[%key:component::vallox::config::step::user::data_description::state::host%]` if I managed to type that correctly.

Reply:
Thanks! The translation compilation complained, but removing `::state it worked (`[%key:component::vallox::config::step::user::data_description::host%]`). 🥳